recipe/meta.yaml

```diff
@@ -56,6 +60,31 @@ outputs:
         convenient.
 
         This package contains utility modules to talk to Avahi with Python easier.
+  - name: avahi
+    script: lib-avahi.sh
+    requirements:
+      build:
+        # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+        # Compilers are named 'c', 'cxx' and 'fortran'.
```

Comment:
Presumable these lines come from some sort of template, but can be removed for a real recipe (please).

```suggestion
```